cmd/geth/flag.go

```diff
@@ -155,6 +155,12 @@ func mustMakeChainIdentity(ctx *cli.Context) (identity string) {
 		cacheChainIdentity = identity
 	}()
 
+	if chainFlagVal := ctx.GlobalString(aliasableName(ChainIdentityFlag.Name, ctx)); chainFlagVal != "" && strings.HasSuffix(chainFlagVal, "/") || strings.HasSuffix(chainFlagVal, "\\") {
```

Comment:
We can use [`path.filepath.Separator`](https://golang.org/pkg/path/filepath/#pkg-constants), or even [`path.filepath.Clean`](https://golang.org/pkg/path/filepath/#Clean) to simplify the code.